.travis.yml

```diff
@@ -7,3 +7,4 @@ rvm:
     - 2.0
     - 2.1
     - 2.2
+    - 2.3.0
```

Comment:
had to specify the `.0` release because of https://github.com/travis-ci/travis-ci/issues/5361